### PR TITLE
[OC-882]: 2.0.0 branch changes for chart version and README files

### PR DESCRIPTION
**Pull request by diptiranjanpx**
Signed-off-by: Diptiranjan

**What this PR does / why we need it**:
2.0.0 has the changes for the release and also changes from the master have been pulled in.

**Test** 
OC-882 has details about install and upgrade details.

### CODE REVIEW

File: px-central/values.yaml [hunk 1]
```diff
@@ -123,17 +123,17 @@ images:
     registry: docker.io
     repo: portworx
     imageName: pxcentral-onprem-ui-frontend-private
-    tag: master
+    tag: 2.0-dev
```
Comment (sharma-tapas) [inline px-central/values.yaml:126]: tag should be 2.0.0-dev

File: px-central/values.yaml [hunk 2]
```diff
@@ -123,17 +123,17 @@ images:
...
     repo: portworx
     imageName: pxcentral-onprem-ui-backend-private
-    tag: master
+    tag: 2.0-dev
   pxcentralMiddlewareImage:
     registry: docker.io
     repo: portworx
     imageName: pxcentral-onprem-ui-lhbackend-private
-    tag: master
+    tag: 2.0-dev
```
Comment (sharma-tapas) [inline px-central/values.yaml:136]: tag should be 2.0.0-dev

File: px-central/values.yaml [hunk 3]
```diff
@@ -123,17 +123,17 @@ images:
...
     repo: portworx
     imageName: pxcentral-onprem-ui-frontend-private
-    tag: master
+    tag: 2.0-dev
   pxcentralBackendImage:
     registry: docker.io
     repo: portworx
     imageName: pxcentral-onprem-ui-backend-private
-    tag: master
+    tag: 2.0-dev
```
Comment (sharma-tapas) [inline px-central/values.yaml:131]: tag should be 2.0.0-dev

File: px-central/values.yaml [hunk 4]
```diff
@@ -170,7 +170,7 @@ images:
     registry: docker.io
     repo: portworx
     imageName: px-backup-base
-    tag: master
+    tag: 2.0-dev
```
Comment (sharma-tapas) [inline px-central/values.yaml:173]: tag should be 2.0.0-dev

**Review by sharma-tapas — CHANGES_REQUESTED** [review]
Our image tags should be 2.0.0 we follow the semVersion way.

**Comment by diptiranjanpx** [comment]
For the tags, need to check with respective component owners.

**Review by sharma-tapas — APPROVED** [review]

**Pull request merged**